pkg/api/controllers/querystrings/keys.go

```diff
@@ -0,0 +1,8 @@
+package querystrings
```

Comment:
I don't think we need an additional package.
In the long run, i think we should merge all packages under pkg/api and unexpose all api related stuff (like query params)

Reply:
Ok, just changed it